src/runtime_src/core/edge/user/aie/common_layer/adf_runtime_api.cpp

```diff
@@ -0,0 +1,654 @@
+/**
+* Copyright (C) 2021 Xilinx, Inc
+*
+* Licensed under the Apache License, Version 2.0 (the "License"). You may
+* not use this file except in compliance with the License. A copy of the
+* License is located at
+*
+*     http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+* WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+* License for the specific language governing permissions and limitations
+* under the License.
+*/
+
+#include "adf_runtime_api.h"
+#include "adf_api_message.h"
+
+#include <algorithm>
+#include <sstream>
+#include <map>
+
+extern "C"
+{
+#include "xaiengine.h"
+}
+
+namespace adf
+{
+
+/********************************* Statics & Constants *********************************/
+
+static constexpr short INVALID_TILE_COORD = 0xFF;
+
+static constexpr int ACQ_WRITE = 0;
+static constexpr int ACQ_READ = 1;
+static constexpr int REL_READ = 1;
+static constexpr int REL_WRITE = 0;
+
+static constexpr unsigned LOCK_TIMEOUT = 0x7FFFFFFF;
+
+
+/********************************* config_manager *************************************/
+
+XAie_DevInst* config_manager::s_pDevInst = nullptr;
+bool config_manager::s_bInitialized = false;
+size_t config_manager::s_num_reserved_rows = 0;
+bool config_manager::s_broadcast_enable_core = false;
+
+err_code config_manager::initialize(XAie_DevInst* devInst, size_t num_reserved_rows, bool broadcast_enable_core)
+{
+    if(!s_bInitialized)
```

Comment:
Since this is in common layer, I think it is OK. But if it is possible, please
1. add as space after key word. e.g. if ()
2. use if () {
     }
 3. same thing for for () {
     }

Reply:
if( has been fixed.